packages/diagrams/frontend/sirius-components-diagrams/src/renderer/palette/group-tool/GroupPalette.tsx

```diff
@@ -0,0 +1,117 @@
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import Paper from '@material-ui/core/Paper';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import { makeStyles } from '@material-ui/core/styles';
+import TonalityIcon from '@material-ui/icons/Tonality';
+import VisibilityOffIcon from '@material-ui/icons/VisibilityOff';
+import { memo, useCallback, useState } from 'react';
+import { useOnSelectionChange } from 'reactflow';
+import { GroupPaletteProps } from './GroupPalette.types';
+import { ContextualPaletteStyleProps } from '../Palette.types';
+import { PalettePortal } from '../PalettePortal';
+import { useHideDiagramElements } from '../../hide/useHideDiagramElements';
+import { useDiagramElementPalette } from '../useDiagramElementPalette';
+import { useFadeDiagramElements } from '../../fade/useFadeDiagramElements';
+import { PinIcon } from '../../../icons/PinIcon';
+import { usePinDiagramElements } from '../../pin/usePinDiagramElements';
+
+const usePaletteStyle = makeStyles((theme) => ({
+  palette: {
+    border: `1px solid ${theme.palette.divider}`,
+    borderRadius: '2px',
+    zIndex: 2,
+    position: 'fixed',
+    display: 'flex',
+    alignItems: 'center',
+  },
+  paletteContent: {
+    display: 'grid',
+    gridTemplateColumns: ({ toolCount }: ContextualPaletteStyleProps) => `repeat(${Math.min(toolCount, 10)}, 36px)`,
+    gridTemplateRows: '28px',
+    gridAutoRows: '28px',
+    placeItems: 'center',
+  },
+  toolIcon: {
+    color: theme.palette.text.primary,
+  },
+}));
+
+export const GroupPalette = memo(({ x, y, isOpened }: GroupPaletteProps) => {
+  const { hideDiagramElements } = useHideDiagramElements();
+  const { fadeDiagramElements } = useFadeDiagramElements();
+  const { pinDiagramElements } = usePinDiagramElements();
+  const { hideDiagramElementPalette } = useDiagramElementPalette();
+  const [selectedElementIds, setSelectedElementIds] = useState<string[]>([]);
+
+  const onChange = useCallback(({ nodes }) => {
+    if (nodes.filter((node) => node.selected).length > 1) {
+      setSelectedElementIds(nodes.filter((node) => node.selected).map((node) => node.id));
+    } else {
+      setSelectedElementIds([]);
+    }
+  }, []);
+  useOnSelectionChange({
+    onChange,
+  });
+
+  const toolCount = 3;
+  const classes = usePaletteStyle({ toolCount });
+
+  const shouldRender = selectedElementIds.length > 1 && isOpened && x && y;
+  if (!shouldRender) {
+    return null;
+  }
+  hideDiagramElementPalette();
+
+  return (
+    <PalettePortal>
+      <Paper className={classes.palette} style={{ position: 'absolute', left: x, top: y }} data-testid="GroupPalette">
+        <div className={classes.paletteContent}>
+          <Tooltip title="Hide elements">
```

Comment:
I see in `Palette.tsx` that `Hide elements` is also used as label for this action, but it should be `Hide element`.
Could you fix that in this PR please and make sure the cypress tests are also updated accordingly?
Also do the same for `Fade`.

Reply:
done